src/library/components/Search.tsx

```diff
@@ -112,13 +111,8 @@ const Search = forwardRef(
           break;
 
         case 'Enter':
-          // TODO: ADS-95 find another solution if we don't want react-router-dom (v6) a dependency of Anatomy due to the use of navigate(), and address when we split lib from docs.
```

Comment:
🎉 
let's make sure to close out this ticket

Reply:
I don't know where that ticket is anymore 😬